ldap/servers/slapd/back-ldbm/vlv_srch.c

```diff
@@ -514,6 +513,8 @@ void
 vlvIndex_init(struct vlvIndex *p, backend *be, struct vlvSearch *pSearch, const Slapi_Entry *e)
 {
     struct ldbminfo *li = (struct ldbminfo *)be->be_database->plg_private;
+    const char *file_suffix = dblayer_get_db_suffix(be);
```

Comment:
Compiler complains  warning: initialization of ‘const char *’ from ‘int’ makes pointer from integer without a cast. I guess its in the case of the ternary operator returning NULL.

Reply:
Thanks James for having noticed the warning that I overlooked !
I am afraid that your guess is wrong: function prototype was missing because I forgot to commit proto-back-ldbm.h so function was assumed to return int ! -;)
  I just pushed a new commit to solve both issues.

Reply:
@progier389 , sorry I can not see the new commit ? and where file_suffix != NULL is tested